Koffee/MainController.fs

```diff
@@ -88,13 +88,15 @@ module MainLogic =
         }
 
         let sortList fsReader field (model: MainModel) = result {
+            let selected = model.SelectedNode.Name
             let desc =
                 match model.Sort with
                 | f, desc when f = field -> not desc
                 | _ -> field = Modified
             model.Sort <- field, desc
             do! refresh fsReader model
```

Comment:
It would be easier to replace this call to refresh with openPath and use SelectName, which will do the selection for you. This should eliminate the need to save the selected name as well.

Reply:
Okay I've reworked this to use openPath. This resulted in the scrolling-out-of-view issue. This is resolved using an event handler on the view.